pkg/rpc/core/mempool.go

```diff
@@ -232,34 +152,25 @@ func BroadcastTxCommit(ctx *rpctypes.Context, tx cmttypes.Tx) (*ctypes.ResultBro
 	case <-deliverTxSub.Canceled():
 		var reason string
 		if deliverTxSub.Err() == nil {
-			reason = "node exited"
+			reason = "Tendermint exited"
```

Comment:
nit: avoid "Tendermint" as the name is not clear in this context. It is actually CometBFT under the hood.
How about some generic "BFT engine exited"?